apps/site/next.config.mjs

```diff
@@ -13,44 +13,61 @@ const nextConfig = {
   // is being built on a subdirectory (e.g. /nodejs-website)
   basePath: BASE_PATH,
   // Vercel/Next.js Image Optimization Settings
-  images: {
-    // We disable image optimisation during static export builds
-    unoptimized: ENABLE_STATIC_EXPORT,
-    // We add it to the remote pattern for the static images we use from multiple sources
-    // to be marked as safe sources (these come from Markdown files)
-    remotePatterns: [
-      {
-        protocol: 'https',
-        hostname: 'avatars.githubusercontent.com',
-        port: '',
-        pathname: '/**',
-      },
-      {
-        protocol: 'https',
-        hostname: 'bestpractices.coreinfrastructure.org',
-        port: '',
-        pathname: '/**',
-      },
-      {
-        protocol: 'https',
-        hostname: 'raw.githubusercontent.com',
-        port: '',
-        pathname: '/nodejs/**',
-      },
-      {
-        protocol: 'https',
-        hostname: 'user-images.githubusercontent.com',
-        port: '',
-        pathname: '/**',
-      },
-      {
-        protocol: 'https',
-        hostname: 'website-assets.oramasearch.com',
-        port: '',
-        pathname: '/**',
-      },
-    ],
-  },
+  images:
```

Comment:
As a follow-up, I think we should make an `apps/site/vercel` folder and `apps/site/cloudflare` folder (well, it'll already exist, see my other comment) for the specific logic. Then, the config file itself will be simple, and call things like `getCloudflareImageConfiguration()` and `getVercelImageConfiguration()` or something?

wdyt @nodejs/nodejs-website